main/drivers/json_test_driver.py

```diff
@@ -0,0 +1,30 @@
+from main.common.IO.json_reader import Reader
```

Comment:
This is fine for now but you should learn to create unit tests.  This way when someone starts up the code they can run all unit tests and they will pass or fail accordingly.  Right now you are not testing if you are getting the right output, just if it runs.